### PR TITLE
manual quickstart

### PR DESCRIPTION
**Pull request by LoganWright**

### CODE REVIEW

File: getting-started/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+currentMenu: getting-started-quick-start
+---
+
+# Vapor Quickstart
+
+This document assumes that the appropriate version of Swift is installed for Vapor. Right now, this is visible on the [vapor](https://github.com/qutheory/vapor#-environment) section of the repository.
```
Comment (tannernelson) [inline getting-started/quickstart.md]: Can we put a small note here about downloading templates with the Toolbox and a link?

File: getting-started/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+#### Package.swift
+
+```Swift
+import PackageDescription
+
+let package = Package(
+    name: "HelloVapor",
+    dependencies: [
+        .Package(url: "https://github.com/qutheory/vapor.git", majorVersion: 0, minor: 15)
```
Comment (tannernelson) [inline getting-started/quickstart.md]: Should we put a placeholder for 15 like `<current_version_here>` or something so we don't forget about it.

Comment (LoganWright) [inline getting-started/quickstart.md]: Yea, I'll just put it that way w/ a link to releases to get latest version.

Comment (tannernelson) [inline getting-started/quickstart.md]: We can probably do a find and replace for 15 -> 16 when 0.16 comes out. So maybe it's not a problem to specifically reference 15 being the current version.

Comment (LoganWright) [inline getting-started/quickstart.md]: It is kind of nice to have it there in a way that users can copy pasta easily.

File: getting-started/quickstart.md (hunk 1 above)
Comment (LoganWright) [inline getting-started/quickstart.md]: Done.

File: getting-started/quickstart.md (hunk 2 above)
Comment (LoganWright) [inline getting-started/quickstart.md]: I left 15 for convenience, but I added a little blurb to check releases regularly 👍

**Pull request merged**